public/app/features/geo/utils/frameVectorSource.ts

```diff
@@ -32,4 +32,26 @@ export class FrameVectorSource extends VectorSource<Geometry> {
     // only call this at the end
     this.changed();
   }
+
+  updateLineString(frame: DataFrame) {
+    this.clear(true);
+    const info = getGeometryField(frame, this.location);
```

Comment:
Path visualization is only works with `coordinates` mode since it requires each point coordinates.